.github/pull_request_template.md

```diff
@@ -5,4 +5,4 @@
 
 ## **Discord contact info**
 <!--- formatted as name#numbers, e.g. PikalaxALT#5823 -->
-<!--- Contributors must join https://discord.gg/d5dubZ3 -->
\ No newline at end of file
+<!--- Contributors must join https://discord.gg/d5dubZ3 -->
```

Comment:
don't put unrelated minor changes in the pr

Reply:
I do not know how to take this change out of the pr. When I analyze the two lines, they seem to be identical, so I don't see why there's a change here in the first place.

Reply:
it looks like a space at the end was deleted i think

Reply:
I added a space, and the space was marked as a change.